test/url-generation.js

```diff
@@ -263,11 +250,11 @@ describe("URL generation", function () {
         const url = imagekit.url({
             path: "/test_path.jpg",
             transformation: [{
-                overlayImage: ""
+                effectShadow: ""
```

Comment:
It should match with convention of passing `true` if I am not wrong. Check how SDK works with other effect parameters that don't necessarily need a value.

Reply:
type definition of `effectShadow ` is the same as other effect parameters. I have removed it from the above test suit and we have a test suit with `true` as its value